meetings/2024/2024-05-15.md

```diff
@@ -5,21 +5,21 @@
 > [!NOTE]
 > Add a ✓ to the `Present` column in the table below, and add anyone missing. Also add to [the template](../template.md) and [GitHub team](https://github.com/orgs/tc39/teams/tg3) if the individual is a regular attendee.
 
-| Present | Name             | GH Username     | TLA | Affiliation       |
-| :-----: | ---------------- | --------------- | --- | ----------------- |
-|    ✓    | Chip Morningstar | @FUDCo          | CM  | Agoric            |
-|    ✓    | Daniel Veditz    | @dveditz        | DPV | Mozilla           |
-|    ✓    | Guy Bedford      | @guybedford     | GB  | OpenJS Foundation |
-|    ✓    | Jasvir Nagra     | @jasvir         |     | Picovisor         |
-|    ✓    | Jordan Harband   | @ljharb         | JHD | HeroDevs          |
-|    ✓    | Jovonni          |                 |     |                   |
-|    ✓    | Kris Kowal       | @kriskowal      | KKL | Agoric            |
-|    ✓    | Mark Miller      | @erights        | MM  | Agoric            |
-|    ✓    | Mathieu Hofman   | @mhofman        | MHN | Agoric            |
-|    ✓    | Nicolò Ribaudo   | @nicolo-ribaudo | NRO | Igalia            |
-|    ✓    | Richard Gibson   | @gibson042      | RGN | Agoric            |
-|    ✓    | Shay Lewis       | @shaylew        |     |                   |
-|    ✓    | Ulises Gascón    | @UlisesGascon   |     | One Beyond        |
+| Present | Name                   | GH Username     | TLA | Affiliation       |
+| :-----: | ---------------------- | --------------- | --- | ----------------- |
+|    ✓    | Chip Morningstar       | @FUDCo          | CM  | Agoric            |
+|    ✓    | Daniel Veditz          | @dveditz        | DPV | Mozilla           |
+|    ✓    | Guy Bedford            | @guybedford     | GB  | OpenJS Foundation |
+|    ✓    | Jasvir Nagra           | @jasvir         |     | Picovisor         |
+|    ✓    | Jordan Harband         | @ljharb         | JHD | HeroDevs          |
+|    ✓    | Jovonni Smith-Martinez | @Jovonni        | JSM | Agoric            |
+|    ✓    | Kris Kowal             | @kriskowal      | KKL | Agoric            |
+|    ✓    | Mark Miller            | @erights        | MM  | Agoric            |
+|    ✓    | Mathieu Hofman         | @mhofman        | MHN | Agoric            |
+|    ✓    | Nicolò Ribaudo         | @nicolo-ribaudo | NRO | Igalia            |
+|    ✓    | Richard Gibson         | @gibson042      | RGN | Agoric            |
+|    ✓    | Shay Lewis             | @shaylew        |     |                   |
+|    ✓    | Ulises Gascón          | @UlisesGascon   |     | One Beyond        |
```

Comment:
```suggestion
|    ✓    | Ulises Gascón          | @UlisesGascon   |  UG  | One Beyond        |
```

Reply:
@UlisesGascon I always remove these when you add them because you technically don't have [a TLA](https://github.com/tc39/notes/blob/main/delegates.txt).  you would get one at the point when you become a delegate or invited expert.  FYI: it would likely be `UGN`